domain/src/main/java/com/ameri/domain/repository/HomeRepository.kt

```diff
@@ -1,6 +1,6 @@
-package com.ameri.newsapi.data.repository
+package com.ameri.domain.repository
 
-import com.ameri.newsapi.data.network.ApiServices
+import com.ameri.data.network.ApiServices
 import javax.inject.Inject
 
 class HomeRepository @Inject constructor(private  val apiServices: ApiServices) {
```

Comment:
Create an interface for this class and put that interface into the domain module (Checkout the ComposeNews's repositories)